examples/website/textures/components/compressed-texture.js

```diff
@@ -111,7 +109,7 @@ export default class CompressedTexture extends PureComponent {
   }
 
   setupBasisLoadOptionsIfNeeded() {
-    if (this.state.supportedFormats.DTX) {
+    if (this.state.supportedFormats.DXT) {
```

Comment:
Maybe the basis loader should be able to do some automatic determination. Will think about that.